auth_api_key_server_env/models/auth_api_key.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2018 ACSONE SA/NV
+# Copyright 2021 Camptocamp SA
+# @author: Simone Orsi <simone.orsi@camptocamp.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+
+from odoo import models
+
+
+class AuthApiKey(models.Model):
+
+    _name = "auth.api.key"
+    _inherit = ["auth.api.key", "server.env.mixin"]
+
+    def _server_env_section_name(self):
```

Comment:
I would drop this and use `server.env.techname.mixin` to not rely on the name which can be used for something more meaningful and not screwed by occasional editing. WDYT?

Reply:
It makes sense, I made the change and added a migration script.